test/Spec.hs

```diff
@@ -124,6 +125,9 @@ tests docker0 = testGroup "Tests"
   , testGroup "SQLite"
       [ testProperty "Parallel" prop_parallel_sqlite
       ]
+  , testGroup "Rqlite"
+      [ testProperty "parallel" $ withMaxSuccess 10 $ prop_parallel_rqlite (Just Weak)
```

Comment:
Could we please add a comment on why the sequential property isn't included here?